debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/util/TimestampUtils.java

```diff
@@ -53,24 +59,55 @@ public final class TimestampUtils {
     public static Instant convertTimestampNoZoneToInstant(String value) {
         final Matcher toTimestampMatcher = TO_TIMESTAMP.matcher(value);
         if (toTimestampMatcher.matches()) {
-            final LocalDateTime dateTime;
             String text = toTimestampMatcher.group(1);
-            if (text.indexOf(" AM") > 0 || text.indexOf(" PM") > 0) {
-                dateTime = LocalDateTime.from(TIMESTAMP_AM_PM_SHORT_FORMATTER.parse(text.trim()));
-            }
-            else {
-                dateTime = LocalDateTime.from(TIMESTAMP_FORMATTER.parse(text.trim()));
-            }
-            return dateTime.atZone(GMT_ZONE_ID).toInstant();
+            return doConvertTimestampNoZoneToInstant(text);
         }
 
         final Matcher toDateMatcher = TO_DATE.matcher(value);
         if (toDateMatcher.matches()) {
-            return LocalDateTime.from(TIMESTAMP_FORMATTER.parse(toDateMatcher.group(1))).atZone(GMT_ZONE_ID).toInstant();
+            String data = toDateMatcher.group(1);
+            return doConvertDateToInstant(data);
         }
 
         // Unable to resolve value
-        return null;
+        return doConvertTimestampNoZoneToInstant(value);
+    }
+
+    private static Instant doConvertTimestampNoZoneToInstant(String text) {
+        final LocalDateTime dateTime;
+        if (text.trim().startsWith("-")) {
```

Comment:
I understand the original code in this method made use of `trim()`.  Is there a way we can minimize that or is that necessary in this context?